### ISSUE TITLE
[question] When will the recent fixes be available on pub.dev ?

### ISSUE THREAD

**Issue by AdamLBS** [googleads/googleads-mobile-flutter#907]
Currently we are running on the last stable version of the plugin (v3.0.0) available on pub.dev.

However we are experiencing the iOS issue where the Validator says "Advertiser assets outside native ad view" when using TemplateType.small (Issue #844 ). As seen on the referenced issue it has been fixed in PR #874 that has been merged 3 weeks ago now. However the fixes are still not available on pub.dev and using a forked repository as a source for the plugin is a little bit janky. 

I would like to know when (and if) theses fixes will be pushed to pub.dev because in the meantime we are stuck using TemplateType.medium to avoid a revenue loss.

Thanks

**Comment by huycozy** [comment]
Labeling this as a documentation issue for the roadmap of the plugin.

/cc @malandr2 for thoughts

**Comment by paco29032023** [comment]
Necesito ayuda

**Comment by mulderpf** [comment]
Instead of documentation, can someone just take action instead.  There are fixes waiting there since June and they need to be published.

**Comment by mulderpf** [comment]
Are there any updates on this?

**Comment by jesusmartinoza** [comment]
Not ideal but remember you can use this to run the latest version:

1. Add the following to your pubspec.yaml
```
  google_mobile_ads:
    git:
      url: https://github.com/googleads/googleads-mobile-flutter
      ref: main
      path: packages/google_mobile_ads
```

2. Remove `ios/Podfile.lock`
3. Run `pod install` inside` /ios` folder

**Comment by AdamLBS** [comment]
Thanks. I knew that we could use this solution but as u said it's not ideal. If the fixes are really stable I don't understand why the package is still not updated on pub.dev

**Comment by mulderpf** [comment]
Thank you, I might try this, but this is NOT ideal as I might end up missing when the updates come out.  I'm starting to wonder if Google is even still supporting this library. 🤷‍♂️

**Comment by mulderpf** [comment]
@huycozy Can you confirm if there will ever be any future updates on this library?  It's been 6 weeks since the last response on this thread and 5 months since the last release of this package.  I am concerned that Google has stopped supporting this package and we are not being told, instead, just radio silence on any further updates.  How much longer do we have to wait?  ANY indication would be useful.

**Comment by malandr2** [comment]
Hi all, 

We have just released version [3.1.0](https://pub.dev/packages/google_mobile_ads/changelog). The latest release includes the fix for https://github.com/googleads/googleads-mobile-flutter/issues/844. 

Thanks all for your patience. We are continuing to work on resolving issues and working on features.

**Comment by mulderpf** [comment]
Why is this marked as completed? None of the fixes we were asking about has been released yet and there has been no more information about when it will be released. Can you at least just give us an indication when everything else will be available?

**Comment by malandr2** [comment]
OP's question was in regard to https://github.com/googleads/googleads-mobile-flutter/issues/844 & https://github.com/googleads/googleads-mobile-flutter/pull/874 which we have addressed, fixed and released in 3.1.0

@mulderpf if you have a question about another fix, please create another issue. Please be as specific as possible when you say "everything else". As far as timelines, we don't have a hard date for the next release as of today but as stated we are continuing to work on resolving issues and working on features.
